rpmlb/builder/koji.py

```diff
@@ -44,7 +44,10 @@ def __init__(
         self.epel = koji_epel
 
         #: Format string for the build target
-        self.target_format = koji_target_format
+        if koji_target_format is None:
+            self.target_format = type(self).DEFAULT_TARGET_FORMAT
```

Comment:
It should, but I was under the impression that you should not use `self` to access class-level attributes (as opposed to instance-level ones).

Unfortunately, I cannot remember where I found it or what was the reasoning behind that – so if you think it is superfluous, I will remove it.